flight/Modules/GPS/GPS.c

```diff
@@ -234,6 +230,9 @@ static void gpsTask(void *parameters)
 	}
 #endif /* PIOS_GPS_MINIMAL */
 
+	timeOfLastUpdateMs = 0;
```

Comment:
OK, I didn't actually make that improvement-- it was already so :)  But moving their declaration.